hack/e2e-test.sh

```diff
@@ -72,15 +72,14 @@ if [ $i -ge ${max_tries} ] ; then
   exit 10
 fi
 
-
-CLOUD_CREDS_DIR="${CLOUD_CREDS_DIR:-/tmp/cluster}"
+CLUSTER_PROFILE_DIR="${CLUSTER_PROFILE_DIR:-/tmp/cluster}"
 
 # Create a new cluster deployment
 export CLOUD="${CLOUD:-aws}"
 export CLUSTER_NAME="${CLUSTER_NAME:-hive-$(uuidgen | tr '[:upper:]' '[:lower:]')}"
 export ARTIFACT_DIR="${ARTIFACT_DIR:-/tmp}"
-export SSH_PUBLIC_KEY_FILE="${SSH_PUBLIC_KEY_FILE:-${CLOUD_CREDS_DIR}/ssh-publickey}"
-export PULL_SECRET_FILE="${PULL_SECRET_FILE:-${CLOUD_CREDS_DIR}/pull-secret}"
+export SSH_PUBLIC_KEY_FILE="${SSH_PUBLIC_KEY_FILE:-${CLUSTER_PROFILE_DIR}/ssh-publickey}"
+export PULL_SECRET_FILE="${PULL_SECRET_FILE:-${CLUSTER_PROFILE_DIR}/pull-secret}"
```

Comment:
If you are going to change the name to `PULL_SECRET_PATH`, then you need to use that name here.

Reply:
I missed this! I'm changing it because of https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh#L16

Reply:
That export in ipi-install-install-commands.sh is isolated to that script. It is not going to conflict with anything in the e2e-test.sh script.

Reply:
Ah, which also means that I led you astray by suggesting that you use `AWS_SHARED_CREDENTIALS_FILE`.

Reply:
Thank you for the quick help! I fixed it. Hope it works